content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -1037,6 +1038,12 @@ Each feature gate is designed for enabling/disabling a specific feature:
   topology of the cluster. See
   [ServiceTopology](/docs/concepts/services-networking/service-topology/)
   for more details.
+- `ServiceIPStaticSubrange`: Enables a strategy for Services ClusterIP allocations, whereby the
+  ClusterIP range is subdivided. Dynamic allocated ClusterIP addresses will be allocated preferently
+  from the upper range allowing users to assign static ClusterIPs from the lower range with a low
+  risk of collision. See
+  [ServiceIPStaticSubrange](/docs/concepts/services-networking/service.md#avoiding-collisions)
```

Comment:
```suggestion
  [ServiceIPStaticSubrange](/docs/concepts/services-networking/service/avoiding-collisions)
```

Reply:
TIL, can you reference sections this way?

Reply:
The md files you see in the git repo will be converted into HTML on a page-by-page basis, so no ".md" would be needed. But my suggestion isn't correct either. It should be:

```
  [ServiceIPStaticSubrange](/docs/concepts/services-networking/service/#avoiding-collisions)
```

Reply:
The text of the hyperlink should be “Avoiding collisions”.